server/uptime-kuma-server.js

```diff
@@ -229,22 +233,20 @@ class UptimeKumaServer {
     /**
      * Get a list of monitors for the given user.
      * @param {string} userID - The ID of the user to get monitors for.
-     * @param {number} monitorID - The ID of monitor for.
+     * @param {number[]} monitorIDs - The IDs of monitors for.
      * @returns {Promise<object>} A promise that resolves to an object with monitor IDs as keys and monitor objects as values.
      *
      * Generated by Trelent
      */
-    async getMonitorJSONList(userID, monitorID = null) {
+    async getMonitorJSONList(userID, monitorIDs = null) {
 
         let query = " user_id = ? ";
-        let queryParams = [ userID ];
 
-        if (monitorID) {
-            query += "AND id = ? ";
-            queryParams.push(monitorID);
+        if (monitorIDs) {
+            query += `AND id IN (${monitorIDs.join(",")}) `;
```

Comment:
Please see https://wikipedia.org/wiki/SQL_injection why parametrisation as in the previous code is nessesary

Reply:
Updated the code using the method you recommended.